examples/sandbox/index.ts

```diff
@@ -1136,6 +1147,17 @@ const $thorchainLiquidityAmount = $("#thorchainLiquidityAmount");
 const $thorchainLiquidityPoolAddress = $("#thorchainLiquidityPoolAddress");
 const $thorchainAddLiquidityResults = $("#thorchainAddLiquidityResults");
 
+const $thorchainArkeoBondProvider = $("#thorchainArkeoBondProvider");
+const $thorchainArkeoModProvider = $("#thorchainArkeoModProvider");
+const $thorchainArkeoOpenContract = $("#thorchainArkeoOpenContract");
+const $thorchainArkeoCloseContract = $("#thorchainArkeoCloseContract");
+const $thorchainArkeoClaimContractIncome = $("#thorchainArkeoClaimContractIncome");
+const $thorchainArkeoClaimEth = $("#thorchainArkeoClaimEth");
+const $thorchainArkeoClaimArkeo = $("#thorchainArkeoClaimArkeo");
+const $thorchainArkeoTransferClaim = $("#thorchainArkeoTransferClaim");
+const $thorchainArkeoAddClaim = $("#thorchainArkeoAddClaim");
+const $thorchainArkeoResults = $("#thorchainArkeoResults");
```

Comment:
remove thorchain vernacular

Reply:
This is relevant because it's in the THORChain block, separate from the Arkeo block. We need to be able to sign Arkeo transactions on THORChain, so this is in place to validate that. 

Effectively, this comes down to switching the chain ID, so I could always remove this from the THORChain block and add a chain ID selector to the Arkeo block and accomplish the same thing.

Reply:
Oh I did not make it down far enough to see the arkeo block and though it was copy/pasta issue. And signing of arkeo txs on thorchain is an aspect I was not aware of tbh.

I don't know what would make more sense from an organization standpoint since it is kind of a different and strange use case. As it is the sandbox, I would just leave it as is.